quasar-utils/build.gradle

```diff
@@ -33,7 +33,14 @@ pluginBundle {
 }
 
 dependencies {
+    implementation project(":quasar-utils:javac-plugin")
     testImplementation("org.junit.jupiter:junit-jupiter-api:$junit_jupiter_version")
     testRuntimeOnly("org.junit.jupiter:junit-jupiter-engine:$junit_jupiter_version")
     testImplementation "org.assertj:assertj-core:$assertj_version"
 }
+
+jar {
+    manifest {
+        attributes "version" : archiveVersion.get()
```

Comment:
because my idea was to ship it together with `quasar-utils`, as a separate artifact but as a direct dependency of it, so that people wouldn't have to declare the dependency explicitly

Reply:
I embedded the javac plugin inside `quasar-utils` Gradle plugin jar